WordPress/gutenberg
# [RNMobile] Correct multiple sliders spacings

Ref to gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2590

## Description

Correct spacing in multiple sliders in `Columns` settings.

## How has this been tested?

1. Open draft post
2. Add `Columns` block
3. Press `Cancel` on block variation
4. Open columns settings
5. Expect correct spacings

Check also settings in other blocks (Cover, Button, Image) if looks properly!





## Screenshots 

ios | android
--- | ---
<img width="300" alt="Screenshot 2020-09-02 at 16 28 16" src="https://user-images.githubusercontent.com/22746080/91996533-5e588600-ed39-11ea-9c15-8284953cede8.png"> | <img src="https://user-images.githubusercontent.com/22746080/91996128-ec803c80-ed38-11ea-964b-311d6c2f20af.png" width="300" />


## Types of changes





## Checklist:
